app/src/pages/inside/membersPage/membersGrid/unassignAction/unassignAction.jsx

```diff
@@ -0,0 +1,108 @@
+import { Fragment, PureComponent } from 'react';
+import { connect } from 'react-redux';
+import { injectIntl, intlShape, defineMessages } from 'react-intl';
+import PropTypes from 'prop-types';
+import { GhostButton } from 'components/buttons/ghostButton';
+import { showModalAction } from 'controllers/modal';
+import { fetch } from 'common/utils';
+import { URLS } from 'common/urls';
+import { canAssignUnassignInternalUser } from 'common/utils/permissions';
+import {
+  activeProjectSelector,
+  activeProjectRoleSelector,
+  userAccountRoleSelector,
+  userIdSelector,
+} from 'controllers/user';
+import { showNotification, NOTIFICATION_TYPES } from 'controllers/notification';
+import UnassignIcon from './img/unassign-inline.svg';
+
+const messages = defineMessages({
+  anassignUser: {
+    id: 'AssignAction.anassignUser',
+    defaultMessage: 'User has been unassigned from project!',
+  },
+  btnTitle: {
+    id: 'AssignAction.anassignBtn',
+    defaultMessage: 'Unassign',
+  },
+});
+
+@injectIntl
+@connect(
+  (state) => ({
+    currentUser: userIdSelector(state),
+    projectId: activeProjectSelector(state),
+    projectRole: activeProjectRoleSelector(state),
+    accountRole: userAccountRoleSelector(state),
+  }),
+  { showNotification, showModalAction },
+)
+export class UnassignAction extends PureComponent {
```

Comment:
Why PureComponent?

Reply:
You component is presentational, it's draws a button control. UI controls is not an actions. So I think the better name for your component is UnassignButton.